### DISCUSSION TITLE
Chat Command: Achievements Completed and Collections logged

### DISCUSSION THREAD

**Discussion by Liam9061** [runelite/runelite#15628] [Q&A] [answered]
Would it be possible to add a chat command to show current "Achievements Completed" and "Collections Logged".
Also could add the Combat tasks too but i'm not sure how easy/difficult this could be to add.

For example :
 !AC = Achievements Completed 492/492
And
!CL = Collections Logged: 500/1422
And
!CT = Combat Tasks = 300/421

**Comment by Enriath** [comment, marked as answer]
Duplicate of #14259 & #15178
